src/js/stores/BrandStore.jsx

```diff
@@ -54,8 +62,71 @@ class BrandStore extends EventEmitter {
       },
     });
   }
+ 
+  updateCard(card){
+    $.ajax({
+      url: "https://dev.calligre.com/api/info/card/" + card.id,
```

Comment:
remind me to update the branding store with the new url changes when this gets merged